test/integration/model/find.test.js

```diff
@@ -1024,18 +1024,33 @@ describe(Support.getTestDialectTeaser('Model'), () => {
         username: Sequelize.STRING
       }, { paranoid: true });
 
-      return User.sync({ force: true }).then(() => {
-        return User.bulkCreate([
-          {username: 'Bob'},
-          {username: 'Tobi'}
-        ]);
-      }).then(() => {
-        return User.destroy({ where: {username: 'Tobi'} });
-      }).delay(5000).then(() => {
-        return User.findAll({ paranoid: true });
-      }).then(users => {
-        expect(users.length).to.be.eql(1);
-      });
+
+      if (dialect === 'sqlite') 
```

Comment:
I'm pretty sure it is just that the write (`destory` but with `paranoid` mode on) doesn't complete before the read (`findAll`) on SQLite. It isn't a timeout issue, just an opportunity for `SQLite` to fully register the change. All I meant to say was that lengthening the wait time reduces the brittleness, which I have done, so it should be strong from here on out. It may represent a problem with the `SQLite` syntax (honestly, I think it's just a virtue of SQLite itself), but this PR is to fix the syntax for `MSSQL` on reading over `paranoid` deleted rows. The SQLite fix in my test reliably resolves the issue for me over several tests.

The failure in the last travis ci build seems to be related to this test - https://github.com/sequelize/sequelize/blob/02ea09c731e117d258e523bd0c77d10fdb55f52f/test/integration/data-types.test.js#L45 - far as I can tell unrelated to my PR.

Other failures that I have seen, but I don't believe are related to this PR:
-
 https://github.com/sequelize/sequelize/blob/99d87dfff6c494994239588d0193a30a0093e7df/test/integration/model.test.js#L362
- https://github.com/sequelize/sequelize/blob/02ea09c731e117d258e523bd0c77d10fdb55f52f/test/integration/configuration.test.js#L18
- "no such file or directory, rename 'coverage/lcov.info' -> 'coverage/integration.info'"

Thoughts?